dags/hivemind_etl_helpers/tests/integration/test_gdrive_loader.py

```diff
@@ -189,8 +187,8 @@ def test__load_by_folder_id(self, mock_load_data):
         self.mock_loader = Mock()
```

Comment:
I still don't get what was the usage of `self.mock_loader`. Could you specify the reason?